interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md

```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
 
 ![WinFormsPropertyGrid-ResetValue](../../../../../images/eurd-win-FormsPropertyGrid-ResetValue.png)
 
+## Set Color Properties
+
+You can use Magnifier to set color properties.
+
+![](../../../../../images/eurd-win-magnifier-button.png)
+
+Do the following to use Magnifier:
+
+- Click the Magnifier button.
+- Move the invoked Magnifier on the screen to find the color you want to set.
+
+	![](../../../../../images/eurd-win-magnifier-get-color.png)
+
+	You can zoom out the magnified zone. To do this use one of the following options:
```

Comment:
You can zoom out the magnified zone.
Can we say "You can use the Maginifier to zoom out."

Also change the second sentence to "Use one of the following options to do this:"